.github/workflows/build.yml

```diff
@@ -7,12 +7,79 @@ on:
     branches: [ "master" ]
 
 jobs:
-
-  build:
-
+  code-formatting:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repo
+      uses: actions/checkout@v3
+    - name: Set up node
+      uses: actions/setup-node@v3
+    - name: Install npm packages
+      run: npm ci
+    - name: Run code formatting tests
+      run: ./tests/run-formatting-tests.sh
+  ruby-linting:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repo
+      uses: actions/checkout@v3
+    - name: Build Docker backend
+      run: |
+        cp dev.env .env
+        docker-compose up --build -d backend
+        docker ps -a
+    - name: Run Ruby formatting tests
+      run: 
+        ./tests/run-ruby-linting-tests.sh
```

Comment:
I don't think this test requires a build. You should be able to run both formatting tests in `code-formatting`